package-lock.json

```diff
@@ -5499,6 +5499,13 @@
         "regl": "^1.3.11",
         "to-px": "^1.0.1",
         "typedarray-pool": "^1.1.0"
+      },
+      "dependencies": {
+        "regl": {
+          "version": "1.7.0",
```

Comment:
Hmm... OK this explains why the count is increased by one.
We need a PR to bump `regl` in `gl-text` too.